### PR TITLE
docs-fr | reference | glossary | csi.md

### PR DESCRIPTION
**Pull request by ryanzidago**

### CODE REVIEW

File: content/fr/docs/reference/glossary/csi.md [hunk 1]
```diff
@@ -0,0 +1,21 @@
+---
+title: Interface de Stockage de Conteneurs (CSI)
+id: csi
+date: 2018-06-25
+full_link: /docs/concepts/storage/volumes/#csi
+short_description: >
+    L'Interface de Stockage de Conteneurs (CSI, de l'anglais Container Storage Interface) définit une interface normalisée pour exposer  les systèmes de stockage aux conteneurs.
```
Comment (oussemos) [inline content/fr/docs/reference/glossary/csi.md]: il y un espace en plus
```suggestion
    L'Interface de Stockage de Conteneurs (CSI, de l'anglais Container Storage Interface) définit une interface normalisée pour exposer les systèmes de stockage aux conteneurs.
```

Comment (ryanzidago) [reply content/fr/docs/reference/glossary/csi.md]: Merci. C'est corrigé.

**Comment by xiangpengzhao** [comment]
@oussemos could you please lgtm and approve this PR if it's ready to merge?

**Pull request merged**